Build/libHttpClient.internal.props

```diff
@@ -12,6 +12,11 @@
     <ClCompile>
       <PreprocessorDefinitions>
         $(libHttpClientDefine);
+        HC_TRACE_BUILD_LEVEL=HC_PRIVATE_TRACE_LEVEL_IMPORTANT;
```

Comment:
have `Condition="'$(Configuration)' != 'Debug'"` on the first branch
have `$(libHttpClientDefine);` included in the second branch